cmd/server/main.go

```diff
@@ -32,7 +38,7 @@ func main() {
 }
 
 func run() error {
-	config := NewConfig()
+	config := config.NewConfig()
```

Comment:
```suggestion
	config := config.New()
```
конструктор надо проще называть